app/services/service_provider_updater.rb

```diff
@@ -25,7 +25,8 @@ def idp_url
     def headers
       {
         'X-LOGIN-DASHBOARD-TOKEN' => IdentityConfig.store.dashboard_api_token,
-        'Content-Type' => 'application/json',
+        'Content-Type' => 'gzip/json',
```

Comment:
[per comment here](https://github.com/18F/identity-idp/pull/9074/files#r1303546377) I *think* we can accomplish this if we just add the `Content-Encoding`, no need to update `Content-Type`?

Reply:
we're now using the custom content type to conditionally unzip

Reply:
got it, thx!